WcaOnRails/config/locales/locales.rb

```diff
@@ -86,5 +86,9 @@ module Locales
       "flag_id": "si",
       "name": "Slovensko",
     },
+    "id": {
+      "flag_id": "id",
+      "name": "Bahasa Indonesia",
```

Comment:
@HafizhHD does this mean "Indonesian" or "Indonesian language"? Just asking because these are two words =)

Reply:
It means Indonesian language. It's a common pronunciation for Indonesian people to call it "Bahasa Indonesia" instead of just "Indonesia".
Anyway, I've fixed the translation a little bit.

Thanks :)